toucan_connectors/postgres/utils.py

```diff
@@ -601,3 +601,19 @@
     13741: 'user_mappings',
     13740: '_user_mappings',
 }
+
+
+def create_query_editor_query(database: str):
```

Comment:
Any idea for a better name than `create_query_editor_query` ?

Reply:
I suggest `build_database_model_extraction_query`. Open to ideas :)

Reply:
Sounds good !